website/docs/r/policy_assignment.html.markdown

```diff
@@ -63,7 +63,7 @@ resource "azurerm_policy_assignment" "example" {
 
   parameters = <<PARAMETERS
 {
-  "allowedLocations": {
+  "listOfAllowedLocations": {
```

Comment:
does this also need to be updated above [here](https://github.com/terraform-providers/terraform-provider-azurerm/pull/6428/files#diff-af242adb59cafee47d7689f105331f02L39) and [here](https://github.com/terraform-providers/terraform-provider-azurerm/pull/6428/files#diff-af242adb59cafee47d7689f105331f02R27) too?

Reply:
I think we'll need to change it more places, the policy in the example here above still uses the old name. We might want to change all our custom policies in tests & examples to use the new builtin name?